src/data/general-cs-quiz.ts

```diff
@@ -1826,6 +1826,15 @@ const generalCSQuiz = [
     Explanation:
       "A hash table allows for efficient data retrieval based on a key through the use of a hash function.",
     Link: "https://en.wikipedia.org/wiki/Hash_table"
+  },
+  {
+    "Question": "What is the Halting Problem, and why is it important in theoretical computer science?",
```

Comment:
let's not use the exact wording from this quora answer

https://www.quora.com/What-is-the-halting-problem-and-why-is-it-important-in-theoretical-computer-science

```suggestion
    "Question": "What is the Halting problem in computability theory?",
```